src/Statistics/BinomialConfidenceInterval.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Statistics/BinomialConfidenceInterval.h"
+
+#include <cmath>
+#include <cstdint>
+
+namespace orbit_statistics {
+
+double sqr(double x) { return x * x; }
+
+[[nodiscard]] BinomialConfidenceInterval WilsonBinomialConfidenceIntervalEstimator::Estimate(
```

Comment:
I am wondering if the inheritance is actually needed, or if this should be simply a free/static method. This for sure that depends on the actual use case.

In case you want to pass an Estimator to a class/function, such that at the point of calling the Estimate function we don't want to stick to a specific estimation method, I agree having a class hierarchy for this.
However, if you know it will be only the Wilson method needed at that point, you might consider having just a free function.

Reply:
I also have a feeling that a free function would suffice.

Reply:
I am all for keeping things simple. But even if we agree, will never need anything but the 95%-CI estimated by Wilson method, I think, there should still be a hierarchy for testing purposes. I expect `SamplingReportDataView` to make use of a CI estimator and I would like to have a way to mock the estimator, so the test tests the UI class alone, not in conjunction with the maths involved in the estimator.

Reply:
Sounds like a valid point. Thanks!